src/Const/README.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Const"
+description: "Const Monoid"
+layout: "guide"
+weight: 80
+---
+
+```haskell
+Const c a
+```
+`Const` a unary function which creates a simple Product type that evaluates to
+`c` for all inputs ignoring its right side. 
+`Const` is a `chainable` that will take any value `c` and regardless of the 
+function called on it will return a new `Const` with the value `c`.
+
+### Example
```

Comment:
I would say that this example should not be included. This is more a case for `First`, for the `mconcat` bits.
We may wat to take the last two samples in this example and include them in the second. Consolidating the (2). Have one example here also matches the rest of the docs.

Reply:
I was going to move the second example to be the example for chain, thoughts?